web/src/reducers.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
I recently learned that we don't retroactively update the copyright year on existing files, so this should stay 2018.

Reply:
This needs to stay as 2018 - here it looks like it's being updated 2019.